### PR TITLE
Update kurdish.json

### PR DESCRIPTION
**Pull request by GoRaN909**
Update and correct some characters place with there codes.

### CODE REVIEW

**Comment by patrickgold** [comment]
Thanks for your PR, really appreciate this!

It seems that you accidentally made the indent of the keys 4 instead of 2 spaces, could you please fix this (else the diff just shows that every line changed).

**Comment by GoRaN909** [comment]
> Thanks for your PR, really appreciate this!
> 
> It seems that you accidentally made the indent of the keys 4 instead of 2 spaces, could you please fix this (else the diff just shows that every line changed).

Hi and sorry for late replay also i hae apologies i didn't understand you what you meant can please tell me what's the problem with this edit?

File: app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/kurdish.json [hunk 1]
```diff
@@ -1,64 +1,65 @@
...
+      { "code":  1670, "label": "چ" },
+      { "code":  1581, "label": "ح" },
+      { "code":  1593, "label": "ع", "popup": {
+        "main": { "code": 1551, "label": "؏" }
+      } },
+      { "code":  1576, "label": "ب" },
+      { "code":  1606, "label": "ن" },
+      { "code":  1605, "label": "م" }
+    ]
   ]
```
[the remark this answers is not in the document]
Comment (GoRaN909) [reply app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/kurdish.json]: okay i did some changes i hope you merge now to the branch if you think that have no prpblems :)

**Comment by patrickgold** [comment]
There's still something off with the indentation, but will merge the layout changes nonetheless. When the next change on this file will occur it will be fixed again. Thanks for your PR!

**Pull request merged**